CHANGES.md

```diff
@@ -36,6 +36,8 @@ This change also aligned the authentication architectures between DC/OS Enterpri
 
 * Allow the DC/OS installer to be used when there is a space in its path (DCOS_OSS-4429).
 
+* Fixed `iam-database-restore` script to not leave old database after restore. (DCOS-42407)
```

Comment:
Is not needed because so far this was not released!

Reply:
CC @mhrabovcin please consider removing this; not sure if an editor will notice this in hindsight

Reply:
You are right, from the context of DC/OS Open this is a new script. Should I move this note to EE version?

Reply:
It's OK to do that in a follow-up PR, but we should really not forget it! :)

Reply:
https://jira.mesosphere.com/browse/DCOS-45870